Products/CMFPlone/controlpanel/browser/maintenance.pt

```diff
@@ -105,6 +105,10 @@
                             tal:condition="form_name"
                             tal:content="form_name">Form name</legend>
 
+                    <p>
+                        <strong i18n:translate="text_zope_database_name">Database name:</strong> <span tal:replace="view/dbName" />
```

Comment:
Well - now I grep'd for ``<strong>`` and ``<span>`` in plone.app.locales and it's really done a couple of times. Will change as suggested, @mauritsvanrees !